CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+Implementing and maintaining a protocol stack is a lot of work, therefore any
+help is appreciated, from creating issues, to contributing documentation, fixing
+issues and adding new features.
+
+Please follow the best-practice contribution guidelines as mentioned below when
+submitting any changes.
+
+### Contributions License Agreement
+
+Before contributing, please read the [Contributions License Agreement](./CLA.md).
+Every contributor must sign this agreement before any code is merged into the
```

Comment:
@mfucci Do you want to make it for "any code" or should I adjust it to "any significant/feature adding/potentially intellectual property relevant code"? (But then always needs to be check and defined).

Reply:
I would say any code.
The main goal of the CLA is that if we want to change to license later, or even sell a commercial license, we don't have to reach out to every single contributor.
So even fixing a type change in a comment should require to sign the CLA. I had to do this to fix a comment in Matter codebase.

Reply:
Good so text stays that way.